pkg/services/team/teamapi/api.go

```diff
@@ -53,36 +57,58 @@ func ProvideTeamAPI(
 	return tapi
 }
 
+func TeamUIDResolver(teamService team.Service) web.Handler {
```

Comment:
This can be replaces by `MiddlewareTeamUIDResolver` right?

Reply:
This one is unused and I forgot to remove it, sorry